### PR TITLE
feat(actor): add loss_scale_factor for seq-mean-token-sum-norm mode

### PR DESCRIPTION
**Pull request by szrlee**
Add configurable loss_scale_factor to replace hardcoded divisor in seq-mean-token-sum-norm loss aggregation. Users can set a constant value to ensure consistent normalization throughout training.

- Add loss_scale_factor param to agg_loss() in core_algos.py
- Add loss_scale_factor field to ActorConfig
- Propagate via global_batch_info to policy loss functions
- Update all direct agg_loss calls in trainers and recipes
- Update YAML configs and DrGRPO documentation

### CODE REVIEW

**Comment by ISEEKYAN** [comment]
do you have any experiment results on the effect of setting loss_scale_factor?

**Comment by tongyx361** [comment]
> do you have any experiment results on the effect of setting loss_scale_factor?

Scaling the loss/objective by a constant, such as the horizon, is a common practice in RL algorithms.
Besides, it can interact further with mechanisms such as gradient clipping and ADAM (variants).

Since this PR does not change the default behavior, but only provides better flexibility, I think it can be approved.
